config/config.exs

```diff
@@ -7,7 +7,7 @@ config :breakfast_roulette, BreakfastRoulette.Repo,
   username: System.get_env("PGUSER"),
   password: System.get_env("PGPASSWORD"),
   hostname: System.get_env("PGHOST"),
-  port: "5432",
+  port: "5432"
```

Comment:
We have everything parametrized. Maybe we could include _yet another env variable_.